node/send_http_request/index.js

```diff
@@ -0,0 +1,37 @@
+const axios = require("axios");
+module.exports = async (req, res) => {
+  let url, method, body, headers;
+
+  try {
+    const payload = JSON.parse(req.payload);
+    url = payload.url;
+    method = payload.method || undefined;
+    headers = payload.headers || undefined;
+    body = payload.body || undefined;
+  } catch (err) {
+    console.log(err);
+    throw new Error("Payload is invalid.");
```

Comment:
Let's use `res.json()` with success:false instead of throwing error for consistency.